catalyst/contrib/dl/callbacks/ignore_metric_callback.py

```diff
@@ -0,0 +1,78 @@
+from catalyst.core.callback import Callback, CallbackOrder
+from catalyst.core.runner import IRunner
+
+
+class IgnoreMetricCallback(Callback):
```

Comment:
just to be sure, could you please update docs with this new callbacks & changelog :)
I now it only draft yet :)

Reply:
speaking about architecture and implementation,
what do you think about `CallbackFilterCallback`?
not only with name filtering, but also with some condition one?
```python
callback_filter = CallbackFilterCallback(
  key={"train": "metric-1"}, 
  lambda={"valid": lambda key, value: isinstance(value, MetricCallback)},
  # or, even we can also add something like
  value={"valid": MetricCallback}
)
```

long story shot, with callback filtering, most of the time you have 2 options:
1. filter by name (`key` field)
2. filter by class (`value` field)
3. some very custom condition -> lambda

what do you think about such proposal?

Reply:
moreover, if we would like to think about filtering in a broader sense....
why do we filter only on loader name? during stage, we have 2 fors - epoch and loader one.
In this case, I think, we could filter not only on loader name, but also on epoch index.

nevertheless, still now sure about user friendly API for this :)

Reply:
maybe, only maybe, something like

```python
callback_filter = CallbackFilterCallback(
  condition=lambda epoch, loader: loader == "train",
  filter=lambda key, value: isinstance(value, MetricCallback),
)
```

Reply:
What do you think about `FilterCallback` or `SkipCallback` or `FilterMetricCallback` as a name for a class?
(I think that we can generate a better name without repeated *callback* word)

Also what do you think about use cases for arguments:

```python
callback. = FilterCallback(
    loader0="to_filter",  # base case - ignore callback for loader
    loader1=["to_fitler1", "to_filter2"],  # ignore multiple callbacks
    loader2={"to_filter1": [1, 22, 333], "to_filter2": [1, 2, 3]},  # turn off callbacks for epochs
    loader3=lambda epoch, loader_name, loader_obj: loader_obj,  # filter with custom function
)
```

Reply:
@Ditwoo I think, `FilterCallback` would be good choice :)
First of all, we should not concentrate on Metric-based callbacks only.